src/Components/SmartComponents/Reports/ReportsPage.js

```diff
@@ -92,6 +95,16 @@ const ReportsPage = () => {
         <FileAltIcon color="var(--pf-v5-global--link--Color)" />
     </Icon>;
 
+    const fetchOsStuff = async () => {
+        try {
+            const data = await getOSExposure();
+            setOsExposureData(data.data);
+            setOsExposureModalOpen(true);
+        } catch (error) {
+            return [];
+        }
```

Comment:
What is the reason we are returning an empty array on error? We need at least to report this error with console.error so that the error is not swallowed completely

Reply:
I looked up the API details for `/report/os` and it appears that it has no error response. We could add generic messaging here.

Reply:
It can be any issue, like 403, 500 which is not usually documented in our OpenAPI schemas

Reply:
Perhaps just having
```
console.error(error);
```
is enough